swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -157,36 +274,30 @@ resource function post countries(@http:Payload CovidEntry[] covidEntries)
 
 In this code:
 
-- It is chosen to either accept the entire payload or send back an error. Copying this straightway results in an error, which is expected as the `ConflictingIsoCodesError` type is not defined yet.
-- This resource has a resource argument named `covidEntries` annotated with `@http:Payload`. This means the resource is expecting a payload with the `CovideEntry[]` type. There are two types of records `CovideEntry[]` and `ConflictingIsoCodesError` that will be used as the return values.
+- It is chosen to accept the entire payload or send back an error. Copying this straightway results in an error, which is expected as the `ConflictingIsoCodesError` type is not defined yet.
+- This resource has an argument named `covidEntries` annotated with `@http:Payload`, which means the resource expects a payload with the `CovideEntry[]` type. Two types of records, `CovideEntry[]` and `ConflictingIsoCodesError`, will be used as the return values.
```

Comment:
We can get rid of @http:Payload now.

Also typo in CovideEntry?

Reply:
as the return values -> as the return types

Reply:
Fixed.